kuma/payments/jinja2/payments/management.html

```diff
@@ -0,0 +1,128 @@
+                            <td>02/19</td>
+                        </tr>
+                    </tbody>
+                </table>
+                {% if not cancel_request %}
+                    <form id="id_stripe_management_form" method="post">
+                        {% csrf_token %}
+                        <button class="button negative" type="button" toggle-confirmation>{{_('Cancel your subscription')}}</button>
+                        <div id="delete-confirmation" class="hidden" aria-hidden="true">
+                            <div class="confirmation column-8 alert danger">
+                                <h5>{{_('Are you sure?')}}</h5>
+                                <p>
+                                    {{_('You will have to set up a new subscription if you wish to resume making payments to MDN.')}}
+                                </p>
+                                <div class="button-group">
+                                    <button class="button" type="button" toggle-confirmation>{{_('Cancel')}}</button>
+                                    <button id="id_stripe_cancel_subscription" name="stripe_cancel_subscription" class="button negative" type="submit">{{_('Confirm')}}</button>
+                                </div>
+                            </div>
+                        </div>
+                    </form>
+                {% endif %}
+            </section>
+        {% else %}
+            <section>
+                <h3>{{_('Monthly Subscription')}}</h3>
+                <div class="bordered-info">
+                    <span>
+                        {% trans url=url('recurring_payment_initial') %}
+                            You have no active susbcriptions.
+                            Why not <a href="{{ url }}">set one up?</a>
+                        {% endtrans %}
+                    </span>
+                </div>
+            </section>
+        {% endif %}
+        {% if cancel_request %}
+            {% if cancel_success %}
+                <div class="confirmation column-8 alert success">
+                    <b>{{_('Your monthly subscription has been successfully cancelled')}}</b>
+                </div>
+            {% else %}
+                <div class="confirmation column-8 alert success">
+                    <b>
+                        {% trans url=support_mail_link, email=support_mail, reference_number=request.user.stripe_customer_id %}
+                            There was a problemt cancelling your subscription,
+                            please contact <a href="{{ url }}">{{ email }}</a>
+                            and provide your customer refrence number {{ reference_number }}.
+                        {% endtrans %}
+                    </b>
+                </div>
+            {% endif %}
+        {% endif %}
+    </div>
+{% endblock %}
+
+{% block js %}
+  {%- if contribution_enabled %}
+    <script src="https://checkout.stripe.com/checkout.js"></script>
```

Comment:
Could we `defer` this?

Reply:
Actually we currently don't need checkout on this page. This will change if we want to allow for more payment management options in future. But for now I'll remove it.